### DISCUSSION TITLE
How to create a table based input method in fcitx5

### DISCUSSION THREAD

**Discussion by Thenujan-0** [fcitx/fcitx5#340] [Q&A]
i want to create a table based input method how do i do it?

**Comment by xingwozhonghua126** [comment]
https://github.com/fcitx/fcitx5-table-extra

你可以看这个

**Comment by wengxt** [comment]
Basically you need to use libime_tabledict command and have a corresponding config file. 

You can take a look at file .conf.in and .txt file.
https://github.com/fcitx/fcitx5-table-other
